### PR TITLE
[explorers] update explorers list

### PR DESCRIPTION
**Pull request by drewrothstein**
* Updates explorers to the latest that I could find.
* Alpha sorts the list.
* Favors explorers with HTTPS vs. HTTP.

### CODE REVIEW

**Comment by gre** [comment]
awesome!

File: src/explorers.js [hunk 1]
```diff
@@ -5,32 +5,32 @@ import type { Account, Operation, CryptoCurrencyConfig } from "./types";
...
   digibyte: hash => `https://digiexplorer.info/tx/${hash}`,
   dogecoin: hash => `https://dogechain.info/tx/${hash}`,
+  ethereum: hash => `https://etherscan.io/tx/${hash}`,
   ethereum_classic: hash => `https://gastracker.io/tx/${hash}`,
   ethereum_testnet: hash => `https://ropsten.etherscan.io/tx/${hash}`,
-  ethereum: hash => `https://etherscan.io/tx/${hash}`,
   hcash: hash => `http://explorer.h.cash/tx/${hash}`,
   komodo: hash => `https://kmdexplorer.io/tx/${hash}`,
-  litecoin: hash => `http://explorer.litecoin.net/tx/${hash}`,
+  litecoin: hash => `https://live.blockcypher.com/ltc/tx/${hash}/`,
```
Comment (meriadec) [inline src/explorers.js]: todo: remove trailing slash

Comment (drewrothstein) [reply src/explorers.js]: It works either way but the site adds it just FYI.

Comment (meriadec) [reply src/explorers.js]: ye, it's just for consistency. Thx for your contribution btw! :)

Comment (drewrothstein) [reply src/explorers.js]: Sure thing, here is a PR: https://github.com/LedgerHQ/ledger-live-common/pull/59

**Pull request merged**